src/User/UserMetadataUpdater.php

```diff
@@ -60,9 +59,13 @@ public function whenDiscussionWasStarted(Started $event)
     public function whenDiscussionWasDeleted(DiscussionDeleted $event)
     {
         $this->updateDiscussionsCount($event->discussion);
+        $this->updateCommentsCount($event->discussion);
     }
 
-    private function updateCommentsCount(Post $post)
+    /**
+     * @param \Flarum\Discussion\Discussion|\Flarum\Post\Post $post
+     */
+    private function updateCommentsCount($post)
```

Comment:
Not sure about calling this argument `$post` if it's not always a post. Maybe we should just pass the user in directly.

Reply:
Good point!

Reply:
Apologies for the confusing approval, I clicked the wrong button. That was meant to request changes.

Reply:
No worries!